packages/belos/src/BelosMinresIter.hpp

```diff
@@ -513,10 +518,6 @@ class MinresIter : virtual public MinresIteration<ScalarType,MV,OP> {
       // Apply operator.
       lp_->applyOp (*V, *Y_);
 
-      // Apply shift
-      if (shift != zero)
-	MVT::MvAddMv (one, *Y_, -shift, *V, *Y_);
-
```

Comment:
Why did we remove this?

Reply:
Using shifts in this solver was never completed.  I'm removing dead code to avoid confusion.